src/capitalsFirst.js

```diff
@@ -19,6 +19,19 @@
  */
 function capitalsFirst(str) {
   // write code here
-}
+  let arr = str.split(' ');
+  let arrUpperCase = [];
+  let arrLowerCase = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i][0] >= 'A' && arr[i][0] <= 'Z') {
```

Comment:
do not hardcode strings inside code move it to constants outside of function and name with UPPER_CASE
ex.: const FIRST_LOWERCASE_LETTER = 'a';
Also, you can move `arr[i][0] >= 'A' && arr[i][0] <= 'Z'` to isUppercaseLetter helper function